provisioning_client/samples/prov_dev_client_ll_x509_sample/readme.md

```diff
@@ -0,0 +1,40 @@
+# Azure IoT Provisioning Using X509 Client Certificates
+
+## Build instructions
+
+1. Modify `.\prov_dev_client_ll_x509_sample.c`: update `id_scope`, `registration_id`, 
+   `x509certificate` and `x509privatekey`.
+   (Optionally, you can change configuration regarding transport, global endpoint, etc.)
+
+1. If you are using an OpenSSL engine, uncomment the `#define SAMPLE_OPENSSL_ENGINE` and set it to the
```

Comment:
Yes: while it looks odd, that's actually the proper way of creating a numbered list in Markdown.

Example here (see markdown):
1. <1.> 
1. <still markdown 1., rendered properly as 2.>

Leaving automatic numbering allows adding a new element in the middle without the need to renumber the whole list &rarr; less changes, nicer diffs.

Reply:
This is actually a `.md` feature. If you put 1's for all sections in a numbered list, it will automatically change it to the correct number is sequence. So if you add a new item, you don't have to go down the whole list to `+1` every number.